dvc/command/get_url.py

```diff
@@ -32,7 +32,7 @@ def add_parser(subparsers, parent_parser):
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
     get_parser.add_argument(
-        "url", help="See `dvc import-url -h` for full list of supported URLs."
+        "url", help="See 'dvc import-url -h' for full list of supported URLs."
```

Comment:
@jorgeorpinel I think you are confusing these. So `'` are used for dynamic stuff like `"something '{}'".format(smth)`. \`(backticks) are used for var names, modules, functions, commands. So in this line backtics should stay.

Reply:
I thought you said back ticks are only used in comments in https://github.com/iterative/dvc/pull/2644#issuecomment-544931854 which is why I changed everything to `'` in printed messages. By "dynamic" I thought you were including any commands to be run like `dvc import-url -h`.

Anyway, Again I think what matters is consistency, don't you agree? Right now all messages use `'`, and back ticks are avoided (only present in some comments and docstrings). Would you like to adopt this convention? I can add the note to the contrib guide.

Reply:
> that is for restructuredtext feature for automatic doc generation, which we don't use, so there is no reason to bother with those

Maybe you will start using automatic doc generation some day, so why not consider adopting the needed conventions right now?

P.s. I am almost out of context and maybe my comment is completely irrelevant. Sorry for the confusion if this is the case.

Reply:
@dashohoxha Great point! I think it is too early to bother with those since we aren't even sure we are going to use it or if there are better alternatives.

Reply:
> * use single quotes in printed messages to wrap dynamically generated parts. E.g. `"adding '{}'".format(output_name)`.
> * use backticks for var names, commands, functions etc in comments and printed messages. E.g.

Alright Ruslan, should be all set now. Please review/approve. And thanks for putting up with my OCD! 😋